tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_ldap_trustedCA.yml

```diff
@@ -0,0 +1,111 @@
+- name: Enable TLS enabled LDAP server
+  block:
+  - name: Enable LDAP setup with trusted CA
+    shell: echo "Stopping LDAP"
+    notify:
+      - INC_CONSTANTS
+      - TEMPNSSDB
+
+  - meta: flush_handlers
+
+  - name: Get runtime password for LDAP
+    shell : cat /etc/dirsrv/slapd-{{ topology }}-testingmaster/pwdfile.txt
+    register: ldapPasswd
+
+  - name: Request for LDAP server certificate with our CA
+    pki:
+      cli: "client-cert-request"
+      hostname : "{{ variable.MASTER_HOSTNAME }}"
+      authType: "basicAuth"
+      nssdb : "/etc/dirsrv/slapd-{{ topology }}-testingmaster/"
+      dbpassword: "{{ ldapPasswd.stdout }}"
+      protocol: "http"
```

Comment:
Minor: any reason why http is preferred over https?

Reply:
Can be fixed, No reason as such for using http here

Reply:
Currently i have excluded the playbook changes for another MR. So marking this comment resolved for this MR.
This MR consists changes related to executing existing tests in Fedora32.